<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>354</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Minha Calculadora</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcela 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>46</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Parcela 2:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditResult">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>431</width>
      <height>64</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8pt;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>52</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Resultado:</string>
    </property>
    <property name="buddy">
     <cstring>textEditResult</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>98</x>
      <y>250</y>
      <width>321</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal; text-decoration:none;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;C A L C U L A D O R A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="execButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>151</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Operações</string>
    </property>
    <widget class="QRadioButton" name="radioButtonMult">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Multipilicação</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSoma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Soma</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSub">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Subtração</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonDiv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>83</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Divisão</string>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="textEdit1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="textEdit2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>261</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>545</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuE_xit">
    <property name="title">
     <string>E&amp;xit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Arquivo">
    <property name="title">
     <string>&amp;Arquivo</string>
    </property>
    <addaction name="action_Abrir"/>
    <addaction name="action_Salvar"/>
    <addaction name="separator"/>
    <addaction name="actionSai_r"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="action_Copia"/>
   </widget>
   <addaction name="menu_Arquivo"/>
   <addaction name="menuEdit"/>
   <addaction name="menuE_xit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Abrir">
   <property name="text">
    <string>&amp;Abrir</string>
   </property>
  </action>
  <action name="action_Salvar">
   <property name="text">
    <string>&amp;Salvar</string>
   </property>
  </action>
  <action name="actionSai_r">
   <property name="text">
    <string>Sai&amp;r</string>
   </property>
  </action>
  <action name="action_Copia">
   <property name="text">
    <string>&amp;Copia</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>textEditResult</tabstop>
  <tabstop>radioButtonSoma</tabstop>
  <tabstop>radioButtonSub</tabstop>
  <tabstop>radioButtonMult</tabstop>
  <tabstop>radioButtonDiv</tabstop>
  <tabstop>execButton</tabstop>
  <tabstop>exitButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>exitButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>476</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>287</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
